scripts/build.js

```diff
@@ -0,0 +1,101 @@
+var fs = require('fs');
+var glob = require('glob');
+var path = require('path');
+var req = require('superagent-promise')(require('superagent'), Promise);
+var urlJoin = require('url-join');
+var yaml = require('js-yaml');
+
+var CDN = 'https://unpkg.com/';
+
+// Major versions.
+var AFRAME_VERSIONS = ['0.2.0', '0.3.0'];
```

Comment:
Yeah, and we only want what we consider the *major* versions. So we ignore like 0.3.1. And we start from 0.2.0 since 0.1.x is really old and the component API has changed drastically.